src/test/java/com/commercetools/sync/sdk2/inventories/InventorySyncMockUtils.java

```diff
@@ -0,0 +1,142 @@
+package com.commercetools.sync.sdk2.inventories;
+
+import static java.util.Collections.singletonList;
+import static java.util.concurrent.CompletableFuture.completedFuture;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.commercetools.api.models.channel.Channel;
+import com.commercetools.api.models.channel.ChannelReference;
+import com.commercetools.api.models.channel.ChannelReferenceBuilder;
+import com.commercetools.api.models.channel.ChannelResourceIdentifierBuilder;
+import com.commercetools.api.models.channel.ChannelRoleEnum;
+import com.commercetools.api.models.inventory.InventoryEntry;
+import com.commercetools.api.models.type.CustomFields;
+import com.commercetools.sync.sdk2.services.ChannelService;
+import com.commercetools.sync.sdk2.services.InventoryService;
+import java.time.ZonedDateTime;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import javax.annotation.Nonnull;
+
+public class InventorySyncMockUtils {
+
+  /**
+   * Returns mock {@link io.sphere.sdk.channels.Channel} instance. Returned instance represents
```

Comment:
Please change the links in the comments.

Reply:
https://github.com/commercetools/commercetools-sync-java/commit/37c82ff9880900242f304c60a89bf06e4ede69fd